spec/parameters.yaml

```diff
@@ -105,6 +105,30 @@ UserSegmentPostRequest:
     required:
       - name
       - key
+DestinationPostRequest:
+  name: destinationBody
+  in: body
+  required: true
+  description: Create a new data export destination.
+  schema:
+    type: object
+    properties:
+      name:
+        type: string
+        description: A human-readable name for your data export destination.
+        example: Example Google Pub/Sub Destination
+      kind:
+        type: string
+        description: The data export destination type. Available choices are kinesis, google-pubsub, and mparticle.
+        example: google-pubsub
+      config:
+        type: object
+        description: destination-specific configuration.
```

Comment:
I'm not too familiar with open api, but is it possible to include schema for each different kind? If not, we should at least mention that the configuration must match the kind.

Reply:
Yeah good idea. I will look into this.